core/src/main/java/hudson/Util.java

```diff
@@ -1597,7 +1597,8 @@ public static boolean isAbsoluteUri(@Nonnull String uri) {
     }
 
     /**
-     * Return true iff the parameter does not denote an absolute URI and not a scheme-relative URI.
+     * Return true if the parameter does not denote an absolute URI and not a scheme-relative URI.
+     * @since 2.3 / 1.651.2
```

Comment:
It is actually new API we needed to introduce for a security fix (https://github.com/jenkinsci/jenkins/commit/2ed0c046dfbb2003a17df27c53777e72c6eaff25, https://github.com/jenkinsci/jenkins/commit/e6efae7d0d742f068498366424b0059e9ed42e4c)